Conda pkg: reformat README; add a bit on how tag is constructed (#39)
populationgenomics/hail: conda/README.md

# Conda package

This folder contains a conda recipe to build the `hail` package for
the [`cpg` Anaconda channel](https://anaconda.org/cpg/hail).

Note that there is also a `hail` package in the
[`bioconda` channel](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/hail)
synced with the [official PyPI release](https://pypi.org/project/hail). However, having
a separate conda package in the `cpg` channel allows us to build it against the codebase
in our fork.

> Because we don't control versioning of Hail project, the `cpg` conda package is
> versioned specifically: we append the git commit hash to the official version tag,
> e.g. `0.2.62.dev289c163`.

Reviewer: ```suggestion
We don't control versioning of original Hail project, so our `cpg` conda release name 
is the official version tag appended with the git commit has, e.g. `0.2.62.dev289c163`.
```

[GitHub Actions CI](../.github/workflows/condarise.yaml) is set up to build the package
using this recipe and push it to Anaconda on every push event to the `main` branch in
the
[CPG hail fork](https://github.com/populationgenomics/hail).

> When installing the package, list the `cpg` channel before `bioconda` to prioritize it
> in the channel order:

Reviewer: ```suggestion
When installing the package, list the `cpg` channel before `bioconda` to prioritize
the channel order:
```
